<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Entry Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Employee_ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textInput1">
     <property name="placeholderText">
      <string>Enter value for Input 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textInput2">
     <property name="placeholderText">
      <string>Enter value for Input 2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label3">
     <property name="text">
      <string>Bonus_Amount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textInput3">
     <property name="placeholderText">
      <string>Enter value for Input 3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label4">
     <property name="text">
      <string>Tax_Amount</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="textInput4">
     <property name="placeholderText">
      <string>Enter value for Input 4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start And End Date</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit"/>
   </item>
   <item>
    <widget class="QDateEdit" name="dateEdit_2"/>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
